test/shlibloadtest.c

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2016 The OpenSSL Project Authors. All Rights Reserved.
+ * Copyright 2016-2017 The OpenSSL Project Authors. All Rights Reserved.
```

Comment:
We would prefer just to have the OpenSSL author copyright, but it's not a requirement.  The terms in the CLA let us do our redistribution terms.  Ok?